test/fail_compilation/fail14486.d

```diff
@@ -27,7 +27,7 @@ struct S4b {          nothrow ~this() {}           nothrow delete(void* p) {} }
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail14486.d(44): Error: delete c0 is not @safe but is used in @safe function test1a
+fail_compilation/fail14486.d(44): Error: cannot use 'delete' in @nogc function 'fail14486.test1a'
```

Comment:
My comments applied to all the change to the test suite.

```D
void main () @safe
{
    auto f = new Object();
    delete f;
}
```

Compiles with your P.R., and obviously shouldn't